sdks/python/hatchet_sdk/context/context.py

```diff
@@ -116,6 +119,10 @@ def was_triggered_by_event(self) -> bool:
     def workflow_input(self) -> JSONSerializableMapping:
         return self.input
 
+    @property
+    def lifespan(self) -> Any:
```

Comment:
python ❤️

Reply:
just don't want people to overwrite it lol